frontends/election-manager/src/app.test.tsx

```diff
@@ -580,7 +580,9 @@ test('tabulating CVRs', async () => {
     expect.anything()
   );
 
-  fireEvent.click(getByText('Mark Tally Results as Official'));
+  const markOfficialButton = getByText('Mark Tally Results as Official');
+  await waitFor(expect(markOfficialButton).toBeEnabled);
```

Comment:
This took me a while to parse mentally. Also, in general it's not guaranteed to be safe to do this as `toBeEnabled` could be a class prototype method and calling it this way would not properly bind `this`. I gather that isn't the case as it appears to be working, but it still feels weird.

Reply:
ah, good point! will wrap in a function